skimage/measure/__init__.py

```diff
@@ -45,4 +46,10 @@
            'grid_points_in_poly',
            'shannon_entropy',
            'blur_effect',
+           'pcc',
```

Comment:
A more descriptive function name would be better, e. g., `pearson_correlation_coefficient` or `pearson_corr_coeff`.